README.md

```diff
@@ -13,29 +13,18 @@ You can use the [GOV.UK Docker environment](https://github.com/alphagov/govuk-do
 
 **Use GOV.UK Docker to run any commands that follow.**
 
-### GitHub token
+### Running the app
 
 You will need a GitHub auth token to build the project or run the full test suite,
-otherwise you will find yourself rate limited. You can create one here:
+otherwise you will find yourself rate limited. You can create one [here](https://github.com/settings/tokens/new). It doesn't need any permissions.
```

Comment:
`here` links are a pet peeve of mine - how about this?

```suggestion
You will need to [create a GitHub auth token](https://github.com/settings/tokens/new) to build the project or run the full test suite, otherwise you will find yourself rate limited. Your token doesn't need any permissions.
```

Reply:
OK, I've amended that this time. For next time, could you explain your reasoning? I don't mind either way.

Reply:
Thanks. Basically they're not very accessible: https://granicus.com/blog/why-click-here-links-are-bad/
Screen reader users can navigate pages by links, and so "here" doesn't make sense out of context, whereas "create a GitHub auth token" does. It's also a little bit more scannable to a sighted person, if they know the link to create a GitHub token is somewhere on the page; the link should pop out.

Reply:
Ah, that's really interesting. Thanks for taking the time to explain, I'll incorporate that into my future linking practices.